app/views/question_flow/forms/_employment_status.html.slim

```diff
@@ -1,17 +1,23 @@
 - partner ||= false
 = form_for(@form, url: request.path, method: :put) do |form|
-  = render "shared/heading",
-            header_text: t("question_flow.#{i18n_key}.heading"),
-            tag: (:partner if partner)
+  = render "shared/heading", tag: (:partner if partner)
 
   = form.govuk_error_summary t("generic.error_summary_title")
 
-  = form.govuk_radio_buttons_fieldset :employment_status, legend: nil
-    = form.govuk_radio_button :employment_status, EmploymentStatusForm::EMPLOYMENT_STATUSES[:in_work],
-            label: { text: t("question_flow.#{i18n_key}.in_work_including_self_employed") },
-            hint: { text: t("question_flow.#{i18n_key}.in_work_including_self_employed_description_#{@check.level_of_help}") }
-    = form.govuk_radio_button :employment_status, EmploymentStatusForm::EMPLOYMENT_STATUSES[:unemployed],
-            label: { text: t("question_flow.#{i18n_key}.unemployed") },
-            hint: { text: t("question_flow.#{i18n_key}.unemployed_description") }
+  = form.govuk_radio_buttons_fieldset :employment_status,
+                                      legend: { text: t("question_flow.#{i18n_key}.heading"), size: "xl", tag: "h1" }
+    ruby:
```

Comment:
question - in the instances where we use ruby, it's either because:

- we want, the responses (i.e `@check.level_of_help`) known about by another page in the flow, not exclusively for the end result page?
- it's fiddly to get the labels from the translations/hashes of where they currently are?

Reply:
So not exactly. In these slim files, you can write a line of ruby code either by starting the line with `-` (or `=`) or by indenting it under a `ruby:` statement. The linter suggests that if you have _multiple_ lines of ruby code you need to write, it's best to put them under `ruby:`, and that also makes it easy to write multi-line ruby statements, because slim sometimes gets confused about interpreting line breaks.